packages/mocks/testdata/marker.json

```diff
@@ -1,13 +1,13 @@
 {
     "address": "tp18vmzryrvwaeykmdtu6cfrz5sau3dhc5c73ms0u",
     "coins": [],
-    "public_key": "",
     "account_number": 19,
     "sequence": 0,
     "manager": "tp15rrl3qjafxzlzguu5x29xh29pam35uetkpnnph",
     "accesscontrol": [],
```

Comment:
Think this should be called "permissions" to match the rust type

Reply:
fixed